wiki/Tournaments/TWC/2019/en.md

```diff
@@ -86,40 +86,55 @@ The osu!taiko World Cup 2019 is run by various community members by distributing
 | ![][flag_US] | **United States** | **[Ney](https://osu.ppy.sh/users/5991961)**, [\_cc](https://osu.ppy.sh/users/1656336), [cheese salad](https://osu.ppy.sh/users/6349821), [yea](https://osu.ppy.sh/users/6874332), [Loopy542](https://osu.ppy.sh/users/5468461) |
 | ![][flag_VE] | **Venezuela** | **[Te amo Kazu](https://osu.ppy.sh/users/1368025)**, [\[\_Chichinya\_\]](https://osu.ppy.sh/users/2140739), [\[\_LordEnder\_\]](https://osu.ppy.sh/users/4609767), [My Angel Kazu](https://osu.ppy.sh/users/5136821), [Nutellitah](https://osu.ppy.sh/users/10613075) |
 
------------------------
+---
 
 ![TWC 2019 Bracket](img/bracket_F.png)
 
------------------------
+---
 
-## Match Schedule: Finals
+## Match Schedule: Grand Finals
 
-### Saturday, 18. May 2019
+### Sunday, 26. May 2019
```

Comment:
probably a template thing again, I've never seen a date written like this (applies to others too)
```suggestion
### Sunday, 26 May 2019
```